test/controllers/callbacks_controller_test.rb

```diff
@@ -0,0 +1,70 @@
+class CallbacksControllerTest < ActionDispatch::IntegrationTest
+  GITHUB_INFO = {
+    "id" => "12345",
+    "email" => "user@example.com",
+    "name" => "John",
+    "login" => "test_user"
+  }
+
+  setup do
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:github] = {
+      "uid" => '12345',
+      "provider" => 'github',
+      "credentials" => {"token" => 'my-token'},
+      "extra" => {"raw_info" => GITHUB_INFO},
+      "info" => GITHUB_INFO,
+    }
+    OmniAuth.config.add_camelization 'github', 'GitHub'
+    if OmniAuth.config.respond_to?(:request_validation_phase)
+      OmniAuth.config.request_validation_phase = ->(env) {}
+    end
+
+  end
+
+  teardown do
+    OmniAuth.config.camelizations.delete('github')
+    OmniAuth.config.test_mode = false
+  end
+
+  def stub_members!(members)
+    OmbuLabs::Auth::CallbacksController.class_eval <<-METHODS, __FILE__, __LINE__ + 1
+      alias_method :__old_organization_members, :organization_members
+
+      def organization_members
+        #{members}
+      end
+    METHODS
+    yield
+  ensure
+    OmbuLabs::Auth::CallbacksController.class_eval do
+      alias_method :organization_members, :__old_organization_members
+    end
+  end
```

Comment:
this setup and stub are based on how Devise tests these actions https://github.com/heartcombo/devise/blob/main/test/integration/omniauthable_test.rb